lms/envs/common.py

```diff
@@ -131,7 +131,7 @@
     'SHIB_DISABLE_TOS': False,
 
     # Toggles OAuth2 authentication provider
-    'ENABLE_OAUTH2_PROVIDER': False,
+    'ENABLE_OAUTH2_PROVIDER': True,
```

Comment:
Changes within this file should not be committed - since they were updated only for testing.